### ISSUE TITLE
[Bug] Crash on iOS with elements that have Layout Compression and VoiceOver active

### ISSUE THREAD

**Issue by slang25** [xamarin/Xamarin.Forms#7563]
### Steps to Reproduce

1. Create a page that contains a `StackLayout` with `CompressedLayout.IsHeadless="true"`
2. Turn on VoiceOver
3. Brace for crash

### Expected Behavior
App should continue to function.

### Actual Behavior
`NullReferenceException` thrown.

### Basic Information

- Version with issue: 4.x
- Last known good version: 3.6

I'm working on a fix and will link to this issue.

**Comment by xperseguers** [comment]
I see the suggested bug fix is targeting XF 4.3, do you plan to backport it to 4.2 as well?

**Comment by slang25** [comment]
I'm happy to have it in as many branches that make sense, wasn't too sure of the process. It's very easy to cherry pick this across to all 4.x branches, not sure if that's something for the XF team to do after merging this.

**Comment by xperseguers** [comment]
@slang25 That should first be accepted on the "master" branch (in that case `vNext (4.3.0)` as triaged from @samhouts) I just wonder it the Xamarin team has an official way of marking where it should go as well, or if a bug fix is targeted at multiple releases.

In TYPO3 Core for instance, we are used to have following sample commit message:

```
[BUGFIX] Adapt XX to prevent YY

Longer description

Resolves: #<ticket-number>
Releases: master, 8.7, 7.6
```

Then it's clear that it should be accepted on master first and then backported to 8.7 and 7.6 branches.

**Comment by slang25** [comment]
When I created the PR, initially it was against `master`, but then I saw in the PR template this comment:
- If this is a bug fix, target the branch of the latest stable version (unless the bug is only in a prerelease or master, of course!).

So my understanding from this is that the XF team want it against the latest stable and will pick it across to other branches if they deem it suitable to. @samhouts is that right?

**Comment by samhouts** [comment]
Hi! This bug is a regression on 4.0, so we are happy to accept it in a service release for 4.2, given that 4.2 is the oldest version we're currently supporting. So please target the PR for 4.2, and it will be automatically merged to 4.3 and master. Thank you!

**Comment by slang25** [comment]
Thanks @samhouts, I've retargeted as advised.
